libs/ui/src/global_styles.tsx

```diff
@@ -1,16 +1,141 @@
+import 'normalize.css';
 import { createGlobalStyle } from 'styled-components';
 
+export interface GlobalStlyesProps {
+  isTouchscreen: boolean;
+  legacyBaseFontSizePx?: number;
+  legacyPrintFontSizePx?: number;
+}
+
 /**
  * Common global styling for VxSuite apps.
  *
- * TODO: Flesh out and vary based on selected pre-defined theme.
+ * TODO: Copied from old App.css files in the frontend packages - could probably
+ * use some cleanup and de-duping with the normalize.css styles we're already
+ * importing.
+ *
+ * TODO: Hardcode base64-encoded versions of our font files and reference here,
+ * so that everything's centralized and we don't have to have duplicate
+ * copies in each app's package.
  */
-export const GlobalStyles = createGlobalStyle`
+export const GlobalStyles = createGlobalStyle<GlobalStlyesProps>`
```

Comment:
Love this centralization. I noticed a typo here - `GlobalStylyesProps`. Do you have spell-checker enabled in your environment? That would have at least notified you.

Reply:
Bah, thanks! You know, I thought I did have a spell checker enabled - need to make sure that's working properly